processing/src/main/java/io/druid/segment/filter/Filters.java

```diff
@@ -336,7 +336,7 @@ public static double estimateSelectivity(
   {
     long numMatchedRows = 0;
     for (int i = 0; i < bitmaps.size(); i++) {
-      final ImmutableBitmap bitmap = bitmapIndex.getBitmap(bitmaps.get(i));
+      final ImmutableBitmap bitmap = bitmapIndex.getBitmap(bitmaps.getInt(i));
```

Comment:
Without this change, the loop was a subject for the inspection. It could be replaced with `for (Integer bitmapIndex : bitmaps) { ... }`, but it uses boxing. Actually `get()` is deprecated in `IntList`.